codecov.yml

```diff
@@ -30,3 +30,6 @@ flags:
     paths:
       - front/
     carryforward: true
+
+ignore:
+  - "front/src/tests"
```

Comment:
Missing new line

Reply:
updated with a new line in the end of codecov too